### PR TITLE
Fix missing checkboxes in share dialog

### LINKED ISSUE #19117: Missing checkboxes in share dialog
sharesidebarcheckboxes

Observed on master (c9397dffea9da757d18514e8eb303c3c88756c84)

Possibly introduced by #18735 

@jancborchardt can you have a look ?

### PR DESCRIPTION
**Pull request by Henni**
fixes #19117 

cc @PVince81 @jancborchardt @raghunayyar

### CODE REVIEW

**Comment by karlitschek** [comment]
great :+1:

File: core/js/sharedialogshareelistview.js [hunk 1]
```diff
@@ -27,25 +27,31 @@
 			'        {{/if}}' +
 			'        <span class="username">{{shareWithDisplayName}}</span>' +
 			'        {{#if mailPublicNotificationEnabled}} {{#unless isRemoteShare}}' +
-			'        <label><input type="checkbox" name="mailNotification" class="mailNotification" {{#if wasMailSent}}checked="checked"{{/if}} />{{notifyByMailLabel}}</label>' +
+			'        <input id="mail-{{shareWith}}" type="checkbox" name="mailNotification" class="mailNotification" {{#if wasMailSent}}checked="checked"{{/if}} />' +
+			'        <label for="mail-{{shareWith}}">{{notifyByMailLabel}}</label>' +
 			'        {{/unless}} {{/if}}' +
 			'        {{#if isResharingAllowed}} {{#if sharePermissionPossible}} {{#unless isRemoteShare}}' +
-			'        <label><input id="canShare-{{shareWith}}" type="checkbox" name="share" class="permissions" {{#if hasSharePermission}}checked="checked"{{/if}} data-permissions="{{sharePermission}}" />{{canShareLabel}}</label>' +
+			'        <input id="canShare-{{shareWith}}" type="checkbox" name="share" class="permissions" {{#if hasSharePermission}}checked="checked"{{/if}} data-permissions="{{sharePermission}}" />>' +
```
Comment (schiesbn) [inline core/js/sharedialogshareelistview.js]: one '>' to much at the end of the line. This '>' is visible after the username and in front of the "can share" checkbox

Comment (Henni) [inline core/js/sharedialogshareelistview.js]: @schiesbn thanks!

**Comment by schiesbn** [comment]
One more issue: If you click on the "Share link" checkbox the checkbox disappear for a second, the text "Share link" moves to the left and then the checked checkbox appears again.

Can this be fixed within this PR or is it a different issue?

**Comment by PVince81** [comment]
@schiesbn not here, to be fixed separately. The problem is that it displays a spinner but doesn't hide the checkbox properly. The styling (negative margin) of the spinner makes it move.

**Comment by schiesbn** [comment]
ok, then :+1: for this pr

**Comment by PVince81** [comment]
Spinner issue raised here: https://github.com/owncloud/core/issues/19137

**Comment by jancborchardt** [comment]
:+1: works, thank you @Henni! Would be ace if you can look into https://github.com/owncloud/core/issues/19137 as well.

**Pull request merged**